src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -731,9 +731,10 @@ private function resolvePreviousNodeFromOtherNode(Node $node): ?Node
             return null;
         }
 
-        $nodes = $node->getStartTokenPos() === $startTokenPos
-            ? []
-            : $this->find($currentStmt, static fn (Node $subNode): bool => $subNode->getEndTokenPos() < $startTokenPos);
+        $nodes = $this->find(
```

Comment:
I see, this should compare start token pos agains current stmt start token pos, I will create a new PR for it

Reply:
https://github.com/rectorphp/rector-src/pull/3912